StoreCgnExpirationActivity/table.ts

```diff
@@ -11,14 +11,17 @@ export const insertCgnExpiration = (
   cgnExpirationTableName: NonEmptyString
 ) => (
   fiscalCode: FiscalCode,
+  activationDate: Date,
   expirationDate: Date
 ): TaskEither<Error, TableService.EntityMetadata> =>
   taskify<Error, TableService.EntityMetadata>(cb =>
     tableService.insertOrReplaceEntity(
       cgnExpirationTableName,
       {
         PartitionKey: eg.String(date_fns.format(expirationDate, "yyyy-MM-dd")),
-        RowKey: eg.String(fiscalCode)
+        RowKey: eg.String(
+          JSON.stringify({ fiscalCode, activationDate, expirationDate })
+        )
```

Comment:
Not sure this is a good option, as in JS object keys order are not deterministic, hence you cannot be sure the same object is serialised into the same string (although V8 optimisation will tend to handle them alphabetically, but this cannot be trusted).

A stronger option may be an array: 
```suggestion
        RowKey: eg.String([fiscalCode, activationDate, expirationDate].join("-"))
```

Or just a concatenated string:
```suggestion
        RowKey: eg.String(`${fiscalCode}-${activationDate}-${expirationDate}`)
```